samples/kaspresso-sample/src/androidTest/kotlin/com/kaspersky/kaspressample/device_tests/DeviceAccessibilitySampleTest.kt

```diff
@@ -50,12 +49,12 @@ class DeviceAccessibilitySampleTest : TestCase() {
                     device.targetContext.packageName,
                     SERVICE_CLASS_NAME
                 )
-                assertTrueSafely { isAccessibilityServiceEnabled() }
+                assertTrueSafely(timeoutMs = 60_000L, intervalMs = 1_000L) { isAccessibilityServiceEnabled() }
```

Comment:
Isn't 60 sec to much?

Reply:
Yeah, looks like it's either working or not. I've experienced cases when 60secs wasn't enough